Python/pystate.c

```diff
@@ -59,7 +59,7 @@ static void _PyThreadState_Delete(PyThreadState *tstate, int check_current);
 /*
    The stored thread state is set by PyThraedState_Swap().
```

Comment:
Another here:
```suggestion
   The stored thread state is set by PyThreadState_Swap().
```

And similarly on line 235.

And line 1143 has `sate` -> `state`, 1223 has `Realisically` -> `Realistically`

Reply:
Done. Thanks